config/routes.rb

```diff
@@ -68,6 +68,12 @@
       resources :achievements
       get 'settings' => 'course_settings#index', as: :settings
       patch 'settings' => 'course_settings#update', as: :update_settings
+
+      resources :registrations, only: [:create, :index, :destroy, :update] do
+        collection do
+          post 'destroy_multiple', as: :destroy_multiple
```

Comment:
I think overloading the delete verb is fine. not sure how to get the controllers to accept that, however.